libs/payments/paypal/src/lib/paypal.service.spec.ts

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { faker } from '@faker-js/faker';
+import { Kysely } from 'kysely';
+
+import { CurrencyManager } from '@fxa/payments/currency';
+import { StripeClient, StripeManager } from '@fxa/payments/stripe';
+import { DB, testAccountDatabaseSetup } from '@fxa/shared/db/mysql/account';
+
+import { PayPalClient } from './paypal.client';
+import { PayPalManager } from './paypal.manager';
+import { PayPalService } from './paypal.service';
+import { ResultPaypalCustomerFactory } from './paypalCustomer/paypalCustomer.factories';
+import { PaypalCustomerManager } from './paypalCustomer/paypalCustomer.manager';
+
+describe('PayPalService', () => {
+  let kyselyDb: Kysely<DB>;
+  let paypalClient: PayPalClient;
+  let paypalManager: PayPalManager;
+  let stripeClient: StripeClient;
+  let stripeManager: StripeManager;
+  let paypalCustomerManager: PaypalCustomerManager;
+  let currencyManager: CurrencyManager;
+  let paypalService: PayPalService;
+
+  beforeAll(async () => {
+    kyselyDb = await testAccountDatabaseSetup([
+      'paypalCustomers',
+      'accountCustomers',
+    ]);
+
+    paypalClient = new PayPalClient({
+      sandbox: false,
+      user: faker.string.uuid(),
+      pwd: faker.string.uuid(),
+      signature: faker.string.uuid(),
+    });
+
+    stripeClient = new StripeClient({} as any);
+    stripeManager = new StripeManager(stripeClient);
+    paypalCustomerManager = new PaypalCustomerManager(kyselyDb);
+
+    paypalManager = new PayPalManager(
+      kyselyDb,
+      paypalClient,
+      stripeManager,
+      paypalCustomerManager
+    );
+
+    currencyManager = new CurrencyManager();
+
+    paypalService = new PayPalService(
+      paypalManager,
+      paypalCustomerManager,
+      currencyManager
+    );
```

Comment:
This should all be constructed with NestJS's test dependency injection tooling.

https://docs.nestjs.com/fundamentals/testing#testing-utilities

See an example of it's usage here:

https://github.com/mozilla/fxa/blob/3354f25b44b06326bfa368950d8909fdfd4262d4/libs/shared/contentful/src/lib/contentful.manager.spec.ts#L46

and here:

https://github.com/mozilla/fxa/blob/3354f25b44b06326bfa368950d8909fdfd4262d4/libs/payments/stripe/src/lib/stripe.service.spec.ts#L12

Reply:
Good catch! Most of it was copied over from `paypal.manager.ts`. I meant to update it like I did for `CapabilityManager` in `capability.manager.ts` but totally forgot! I'll make the change!

Reply:
Updated. Please let me know if I need to update anything else. Ty!